src/sanitizer.js

```diff
@@ -66,6 +64,7 @@ const SELF_CLOSING_TAGS = {
   'col': true,
   'hr': true,
   'img': true,
+  'input': true,
```

Comment:
@dvoytenko adding `link` to self-closing tags causes a test to fail:

```bash
Chromium 51.0.2704 (Ubuntu 0.0.0) sanitizeHtml should NOT output security-sensitive markup FAILED
	AssertionError: expected 'a</link>c' to equal 'ac'
	    at Context.<anonymous> amphtml/test/functional/test-sanitizer.js:56:50)
```

This is fine right? We just need to fix the test to drop the closing `</link>` tag no?